src/formatDate.js

```diff
@@ -50,7 +50,49 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const outputDateArray = [];
+  const initialDateArray = date.split(fromFormat[3]);
```

Comment:
```suggestion
  const outputDateParts = [];
  const initialDateParts = date.split(fromFormat[3]);
```
Try not to use data type in var name.

Reply:
Fixed ✔